.devcontainer/Dockerfile

```diff
@@ -69,7 +69,8 @@ RUN echo "Install general purpose packages" && \
         ninja-build \
         perl \
         pkg-config \
-        python2.7 \
+        python3 \
+        python-is-python3 \
```

Comment:
There is a [dependency on Python 3](https://github.com/magma/magma/blob/master/orc8r/gateway/python/python.mk#L24) in the code. And using a focal fossa (=20.04) we here, an upgrade might be necessary.

It might be the case that this change is needed anyway and could be extracted into a small PR which could than actually merged. 🤔